charms/openfga-k8s/src/charm.py

```diff
@@ -419,6 +429,9 @@ def _on_openfga_relation_changed(self, event: RelationChangedEvent):
             event.defer()
             return
 
+        if not self.is_openfga_server_running():
```

Comment:
So is this issue simply because we do some form of relation, join,depart,etc. and we tried to create a store? If so then yes this will fix it